synapse/lib/storm.py

```diff
@@ -896,3 +897,35 @@ def getArgParser(self):
 
         async for node, path in subg.run(runt, genr):
             yield node, path
+
+class TeeCmd(Cmd):
+    '''
+    words go here
+    '''
+    name = 'tee'
+
+    def getArgParser(self):
+        pars = Cmd.getArgParser(self)
+
+        pars.add_argument('--join', '-j', default=False, action='store_true',
+                          help='Emit inbound nodes')
+
+        pars.add_argument('query', nargs='*',
+                          help='Specify a query to execute')
+
+        return pars
+
+    async def execStormCmd(self, runt, genr):
+
+        async for node, path in genr:  # type: s_node.Node, s_node.Path
+            for query in self.opts.query:
+                query = query.strip('{}')
```

Comment:
this would be bad if the query has subqueries. Just `query[1:-1]` is what you want.